speechmatics/client.py

```diff
@@ -72,7 +72,6 @@ def __init__(
             self.connection_settings.set_missing_values_from_config(UsageMode.RealTime)
         self.websocket = None
         self.transcription_config = None
-        self.translation_config = None
```

Comment:
Since this is introducing potentially breaking changes to existing functionality it should be done in a separate MR and released as minor client version? Otherwise anyone interested in using the new audio-events feature could potentially be impacted.

Reply:
Can I not just change the minor version on here? The audio-events feature will be on top of that bc breaking change regardless?